### DISCUSSION TITLE
Merged branches in web application

### DISCUSSION THREAD

**Discussion by alexVitale75** [community/community#37610] [Other Feature Feedback, Questions, & Ideas]
Hello there, 
I'm having this issue:
whenever I merge a branch into devel, without using a pull request (e.g. from a desktop client, or using cli command), and then of course I push devel branch, I cannot see it as "merged" in the branches list (web ui).
Is this normal? I mean: there's no other way than using pull request, in order to see the branch "merged"?
I'm coming from other services (GutLab, Bitbucket), when the branches were properly shown as "merged", even if the merge was done without making a pull request.
Thanks to anyone who can give me some help!

**Comment by jfullstackdev** [comment]
you mean direct push ? 

as for me, I'm using GitHub branches to create a pull request, this way, I can easily remember anything I've done on the codebase, even if that will be extra clicks away
